packages/make-styles/src/runtime/resolveStyleRules.test.ts

```diff
@@ -394,7 +390,7 @@ describe('resolveStyleRules', () => {
           animationDuration: '5s',
         }),
       ).toMatchInlineSnapshot(`
-        @-webkit-keyframes f13owpa8 {
+        @-webkit-keyframes f1q8eu9e {
```

Comment:
Hashes of keyframes have been changed because of a change in `cssifyObject()` that always adds trailing commas now.